code/game/objects/structures/crates_lockers/closets/secure/medical.dm

```diff
@@ -275,3 +267,29 @@
 		new /obj/item/weapon/reagent_containers/glass/bottle/reagent/acetone(src)
 		new /obj/item/weapon/reagent_containers/glass/bottle/reagent/acid(src)
 		new /obj/item/weapon/reagent_containers/glass/bottle/reagent/diethylamine(src)
+
+
+/obj/structure/closet/secure_closet/coroner
+	name = "coroner's locker"
+	icon_state = "securecoroner1"
+	icon_closed = "securecoroner"
+	icon_locked = "securecoroner1"
+	icon_opened = "securecoroneropen"
+	icon_broken = "securecoronerbroken"
+	icon_off = "securecoroneroff"
+	req_access = list(access_morgue)
+
+	New()

```

Comment:
This should be `obj/structure/closet/secure_closet/coroner/New()` and indented as such instead of being part of the variable defines for `obj/structure/closet/secure_closet/coroner`

Reply:
Im sorry, but what is the difference between doing that and not following the same formatting like the other lockers above? I can do it your way, but then I'm going to refactor the entire page to include that for all the lockers.

Reply:
The difference is that none of the maintainers like it. It's part of the guidelines apparently for coding, give all procs and such a direct path so to make searching for them easier. You won't be able to search for `/obj/item/weapon/dildo/New()` if it was indented like above.
Personally I'd like the object/file structure to be like direct pathing so you could indent, just follow the file structure to find the object, but everything is kind of set in stone as it is. Best to keep to the format so less people scream at you

Reply:
New additions should follow absolute pathing, relative pathing is basically impossible to use grep against, which makes finding specific code quite difficult. You are not obligated to change any of the other lockers to follow this, but you can do so if you wish.